src/main/java/io/github/bookcrawler/core/BookStore.java

```diff
@@ -61,6 +63,33 @@ public BooksLinkCrawler crawler() {
         public BookInfoParser parser() {
             return new PacktBookInfoParser();
         }
+    },
+    HELION {
+        @Override
+        public String startUrl() {
+            return "http://helion.pl/promocja-dnia";
+        }
+
+        @Override
+        public String domainUrl() {
+            return "http://helion.pl";
+        }
+
+        @Override
+        public BookExtractor extractor() {
+            return new HelionBookExtractor(new JsoupSourceScrapper());
+        }
+
+        @Override
+        public BooksLinkCrawler crawler() {
+            //empty, not used
+            return null;
```

Comment:
It will be gone when Spring comes in that code. We will be injecting crawler only when there is need of it.